distribution/lib/Standard/Base/0.0.0-dev/src/Runtime.enso

```diff
@@ -52,8 +53,20 @@ get_stack_trace =
 gc : Nothing
 gc = @Builtin_Method "Runtime.gc"
 
+## ADVANCED
+
+   Asserts that the given action succeeds, otherwise throws a panic.
+
+   Assertions are disable by default, meaning that call to this method is
+   a no-op. To enable assertions, set the environment variable
+   `ENSO_ENABLE_ASSERTIONS=true`
```

Comment:
Is the below signature correct, when the `~action` can in fact be also a constant? Shouldn't it be just `Boolean`? Btw. there is a check for suspended lazy arguments since
- #6883 
- #7727